contexts/network-settings.tsx

```diff
@@ -267,8 +287,14 @@ export const NetworkSettingsProvider = ({ children }) => {
     }
   };
 
+  function toLower(str: string) {
```

Comment:
This function is repeated, I'll wager we are using lodash somewhere -- if you really want to use a helper -- use either from lodash or export this function from the `helper/`